### PR TITLE
Audit record uses the exact value of: X-FHIR-FORWARDED-URL header as the URL called for the log entry #2473

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Comment by lmsurpre** [comment]
I was going to suggest that we document that we'll use the request from the servlet context (not the original request url), but it looks like we never actually added the `target` to our CADF documentation.  Maybe it would be good to fix that omission as part of this PR?

**Comment by prb112** [comment]
> I was going to suggest that we document that we'll use the request from the servlet context (and not the original request url), but it looks like we never actually added the `target` to our CADF documentation at https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide#4111-cadf-audit-log-entry. Maybe it would be good to fix that omission as part of this PR?

Sure, once I have a clean workspace, I'll update.

**Comment by prb112** [comment]
All updated

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 1]
```diff
@@ -1429,6 +1429,12 @@ The following table describes the JSON fields of the CADF audit log entries logg
 |`observer/geolocation/city`                           |Value is determined by "fhirServer/audit/serviceProperties/geoCity" configuration property.|
 |`observer/geolocation/state`                          |Value is determined by "fhirServer/audit/serviceProperties/geoState" configuration property.|
 |`observer/geolocation/region`                         |Value is determined by "fhirServer/audit/serviceProperties/geoCounty" configuration property.|
+|`target/id`                                           |Value is the UUID from the resource|
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:1432]: is this an internal UUID or is this the logical id?  if its the logical id, then its not necessarily a UUID.

Comment (prb112) [reply docs/src/pages/guides/FHIRServerUsersGuide.md]: Fixed

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**